src/xtopcom/xstatectx/src/xstatectx_base.cpp

```diff
@@ -35,10 +35,11 @@ void xstatectx_base_t::sync_unit_block(const base::xvaccount_t & _vaddr, uint64_
     xinfo("xstatectx_base_t::sync_unit_block account=%s,end_h=%ld,connect_h=%ld", _vaddr.get_account().c_str(), end_height, latest_connect_height);
 }
 
-xobject_ptr_t<base::xvbstate_t> xstatectx_base_t::load_proposal_block_state(base::xvblock_t* prev_block) const {
+xobject_ptr_t<base::xvbstate_t> xstatectx_base_t::load_proposal_block_state(const base::xvaccount_t & addr, base::xvblock_t* prev_block) const {
```

Comment:
we can use prev_block->get_account() here to replace addr

Reply:
xvaccount type